src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -158,22 +158,18 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "opencv-python-headless": ("cv2",),
     "opensearch-py": ("opensearchpy",),
     # opentelemetry
-    "opentelemetry-api": ("opentelemetry",),
-    "opentelemetry-exporter-otlp": ("opentelemetry.exporter",),
-    "opentelemetry-exporter-otlp-proto-grpc": ("opentelemetry.exporter.otlp.proto.grpc",),
-    "opentelemetry-exporter-otlp-proto-http": ("opentelemetry.exporter.otlp.proto.http",),
```

Comment:
removed these 3, as they map 1:1 with a simple `s/-/./g` (the first one was incorrect.. missing the final `.otlp` part)

Reply:
Btw - the default replacement is `s/-/_/g`, so these break now. We should add these back.

```
pants.backend.python.dependency_inference.rules.UnownedDependencyError: Pants cannot infer owners for the following imports in the target <redacted>:

  * opentelemetry.exporter.otlp.proto.http.trace_exporter.OTLPSpanExporter (line: 6)
  * opentelemetry.sdk.resources.Resource (line: 11)
  * opentelemetry.sdk.resources.SERVICE_NAME (line: 11)
  * opentelemetry.sdk.trace.TracerProvider (line: 12)
  * opentelemetry.sdk.trace.export.BatchSpanProcessor (line: 14)
  * opentelemetry.sdk.trace.export.ConsoleSpanExporter (line: 15)
  * opentelemetry.sdk.trace.export.SpanExporter (line: 16)
```

Reply:
Oh, good catch. Hopefully fixes in #20749

Reply:
doh 🤦🏽